src/scripts/site.js

```diff
@@ -1135,10 +1165,6 @@ var Dashboard = {
         define("filterMenu", [embyWebComponentsBowerPath + "/filtermenu/filtermenu"], returnFirstDependency);
         define("sortMenu", [embyWebComponentsBowerPath + "/sortmenu/sortmenu"], returnFirstDependency);
         define("registrationServices", [embyWebComponentsBowerPath + "/registrationservices/registrationservices"], returnFirstDependency);
-
-        "cordova" === self.appMode || "android" === self.appMode
-            ? define("fileupload", ["cordova/fileupload"], returnFirstDependency)
```

Comment:
why is this dropped? because `fileupload` is for camera sync we dropped, right?

Reply:
Yes